CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- Give a warning when `supports` conditions are used in `@import` statements.
+
```

Comment:
It's not yet clear if we'll be releasing this change; I don't mind writing up the changelogs at release time. Let's just remove this.